scripts/postinstall.js

```diff
@@ -1,54 +1,67 @@
 /* eslint-disable no-console */
-
 const chalk = require('chalk');
 const path = require('path');
 const fs = require('fs-extra');
+const { buildConfig } = require('./config-builder');
 
-const configDirectory = path.join(__dirname, '../config')
-const storybookDirectory = path.join(__dirname, '../.storybook');
-
-// Check if there is a .storybook dir in current project
-console.log(chalk.blue(`Checking for .storybook at ${storybookDirectory}`));
-fs.access(storybookDirectory, fs.constants.W_OK, (err) => {
-  if (err) {
-    console.log(chalk.red(`Could not find .storybook folder at ${storybookDirectory}`));
-    return;
-  }
+// Check if there is a .storybook dir in projectPath
+const currentDir = process.cwd();
```

Comment:
__dirname would be scripts, so in this instance we would want path.join(__dirname, '../'). cwd() gives the same result (without having to do the '../'), though not sure if it may change based on context...?